mods/persistence/modules/cortical_chat/stack_radio.dm

```diff
@@ -0,0 +1,76 @@
+var/const/COR_FREQ = 1551
+
+/obj/item/radio/stack_radio
+	name = "cortical receiver"
+	icon = 'icons/obj/items/device/radio/beacon.dmi'
+	frequency = COR_FREQ
+	canhear_range = 0
+
+	power_usage = 0
+
+// No modification of stack_radios, if they're somehow accessed.
+/obj/item/radio/stack_radio/attack_self()
+	return
+
+// Skipping a lot of checks here since cortical chat is permanently active, for most purposes.
+/obj/item/radio/stack_radio/talk_into(var/mob/living/M, message, var/verb = "says", var/decl/language/speaking = null)
+	if(!M || !message) return 0
+
+	if(speaking && (speaking.flags & (NONVERBAL|SIGNLANG))) return 0
+
+	// Sedation chemical effect prevents cortical chat use as well.
+	var/mob/living/carbon/C = M
+	if ((istype(C)) && (C.chem_effects[CE_SEDATE] || C.incapacitated(INCAPACITATION_DISRUPTED)))
+		to_chat(M, SPAN_WARNING("You're unable to reach \the [src]."))
+		return 0
+
+	if((istype(C)) && C.radio_interrupt_cooldown > world.time)
+		to_chat(M, SPAN_WARNING("You're disrupted as you reach for \the [src]."))
+		return 0
+
+	if(istype(M)) M.trigger_aiming(TARGET_CAN_RADIO) // Cortical chat can't save you here.
+
+	if(!radio_connection)
+		set_frequency(frequency)
+
+	if(loc == M)
+		playsound(loc, 'sound/effects/walkietalkie.ogg', 20, 0, -1)
+
+	if (!istype(radio_connection))
+		return 0
+
+	//#### Tagging the signal with all appropriate identity values ####//
+
+	// ||-- The mob's name identity --||
+	var/displayname = M.name	// grab the display name (name you get when you hover over someone's icon)
+	var/real_name = M.real_name // mob's real name
+	var/voicemask = 0 // the speaker is wearing a voice mask
+
+	// The mob is disguising their identity:
+	if (ishuman(M) && M.GetVoice() != real_name)
```

Comment:
Don't allow disguising.

Reply:
Instead add a verb that allows a character to set their "alias" on cortical chat. By default have it be some random gibberish.